File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        if (len(numbers) == 2):
                return [1, 2]
        leftPointer = 0 
        rightPointer = len(numbers) - 1 
        while (leftPointer < rightPointer):
            sum = numbers[leftPointer] + numbers[rightPointer]
            if (sum == target):
                return [leftPointer+1, rightPointer+1] # because of we should return 1-index array
            elif (sum < target):
                leftPointer += 1
            else:
                rightPointer -= 1

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.js
/**
 * @param {number[]} numbers
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(numbers, target) {
    var left = 0
    var right = numbers.length - 1
    if (numbers.length == 2){return [1, 2]}
    while(left < right){
          if (numbers[left] + numbers[right] == target){
              return [left+1, right+1]
          }else{
              if(numbers[left] + numbers[right] < target){
                  left += 1
              }else{
                  right -= 1
              }
          }
        
          }
};